INSTALL_MAC.md

```diff
@@ -0,0 +1,61 @@
+# Installation instructions for Mac
+
+## Usage
+
+### Install prerequisites
+
+The cross compilation toolchain and docker have to be installed. 
+The following instruction have been tested on Mac OS Mojave (10.14).
+
+```bash
+# Ensure your brew install is healthy
+brew doctor
+brew install cmake git wget curl bash-completion qemu
+python3 -m pip install --user -U  colcon-common-extensions rosdep vcstool
+```
+
+[Install Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/)
+
+### Building a workspace
```

Comment:
With the new implementation approach, most of these steps will be invalid. I think it's better to remove them from here if you're planning to move forward with this PR.

The Mac installation steps can only contain the Mac-specific part and the rest can stay in the main README.